.github/workflows/gradle.yml

```diff
@@ -643,6 +643,43 @@ jobs:
           attempt_limit: 3
           attempt_delay: 5000 # in ms
 
+      - name: Add coverage to PR
+        if: github.event_name == 'pull_request'
+        id: jacoco
+        uses: madrapps/jacoco-report@v1.3
+        with:
+          # todo(mfsiega-airbyte): when v1.4 is released, use wildcards per https://github.com/Madrapps/jacoco-report/issues/20.
+          paths: |
+            ${{ github.workspace }}/airbyte-analytics/build/reports/jacoco/test/jacocoTestReport.xml,
```

Comment:
can we do a glob and make this a little less wordy? e.g. ${{ github.workspace }}/**/build/reports/jacoco/**

Reply:
Sadly not: https://github.com/airbytehq/airbyte/actions/runs/3914203500/jobs/6691026776

Reply:
RIP in peace :(

https://github.com/Madrapps/jacoco-report/issues/20